src/themes/light.scss

```diff
@@ -185,4 +185,22 @@
   --wxc-tooltip-background: #{$wxc-white-alpha-100};
   --wxc-tooltip-border: 1px solid #{$wxc-black-alpha-20};
   --wxc-tooltip-border-color: #{$wxc-black-alpha-20};
+
+  --wxc-input-on--normal--background: #{$wxc-blue-60};
+  --wxc-input-on--hover--background: #{$wxc-blue-70};
+  --wxc-input-on--active--background: #{$wxc-blue-80};
+  --wxc-input-on--disabled--background: #{$wxc-blue-20};
+
+  --wxc-input-on--normal--foreground: #{$wxc-white-alpha-95};
+  --wxc-input-on--disabled--foreground: #{$wxc-white-alpha-80};
+
+  --wxc-input-off--normal--background: #{$wxc-black-alpha-20};
+  --wxc-input-off--hover--background: #{$wxc-black-alpha-30};
+  --wxc-input-off--active--background: #{$wxc-black-alpha-40};
+  --wxc-input-off--disabled--background: #{$wxc-black-alpha-11};
+
+  --wxc-input-off--normal--foreground: #{$wxc-black-alpha-95};
+  --wxc-input-off--disabled--foreground: #{$wxc-black-alpha-40};
+
+  --wxc-input-off--normal--border-color: #{$wxc-black-alpha-50};
```

Comment:
group by state first (normal) and then property (border):
```css
--wxc-input--on--normal--background
--wxc-input--on--normal--border

--wxc-input--on--hover--background
--wxc-input--on--hover--border

--wxc-input--off--normal--background
etc

Reply:
use just `--border` instead of `--border-color`, after all, background and foreground are colors too and we did not suffix them